site/gdocs/Chart.tsx

```diff
@@ -23,6 +24,9 @@ export default function Chart({
         }
     }
 
+    const isExplorer = Url.fromURL(d.url).isExplorer
+    const height = d.height || isExplorer ? 700 : 550
```

Comment:
If we don't want to abstract and automate the height properties of full-width "featured" explorers yet (e.g. https://ourworldindata.org/poverty#explore-data-poverty or https://ourworldindata.org/migration-definition - currently wrongly in the right column), then this should probably be:

```suggestion
    const height = d.height || 575
```

Otherwise, it means that control-less explorers are taller than their grapher chart counterparts (see bottom of [demo doc](https://ptolemy.owid.cloud/admin/gdocs/1MfJkgmcH8-wM0Rd4AcksSrGaw6Hh4Uyrx2aouGge3gM/preview))

Reply:
Good catch!

How about we also check to see if `hideControls=true` and if so, we show the default grapher height?

Reply:
yep, although it might make for an easier ternary to check for `isExplorer && hasControls` which is the only condition for which charts (explorers here) are taller (pending the "featured" feature). Anything else is 575.

But taller explorers also have a few more styles currently:
`min-height: 740px; max-height: 950px; height: 100vh;`

We can deal with this now or consider it beyond the SDG scope and push that back if it needs more discussion.

Reply:
Where are you getting those other rules from? They don't seem to be applying to gdoc-embedded explorers currently.

Reply:
@ikesau these come from full-width explorers, e.g. https://ourworldindata.org/coronavirus#explore-the-global-situation or https://ourworldindata.org/poverty#explore-data-poverty. They are manually added in the WP admin to the style attribute of the iframe embed code, and [honored in the formatting code](https://github.com/owid/owid-grapher/blob/02f02c11388cfe59e258965c256ad4e5f3d568e7/baker/formatWordpressPost.tsx#L432-L433):

https://github.com/owid/owid-grapher/blob/02f02c11388cfe59e258965c256ad4e5f3d568e7/baker/formatWordpressPost.tsx#L432-L433

Reply:
The full-width explorers with controls are not supported on gdocs at the moment. This is currently ok since gdocs is only used for SDGs pages, which don't use those full-width explorers with controls. That's what I meant by:

> We can deal with this now or consider it beyond the SDG scope and push that back if it needs more discussion.